src/main/java/hudson/plugins/jira/listissuesparameter/JiraIssueParameterDefinition.java

```diff
@@ -110,9 +120,19 @@ public String getDisplayName() {
         public final String key;
         public final String summary;
 
-        public Result(final Issue issue) {
+        public Result(final Issue issue, String altSummaryFields) {
```

Comment:
pls add tests to cover that behavior

Reply:
There wasn't much coverage of this file so I added some specific  cases

Reply:
that's breaking backward compat..

Reply:
@olamy As in older builds won't have the second parameter? I'm not an expert in Jenkins plugins would having 2 constructors help?

Reply:
I Switched to DataBoundSetter,